docs/Community/governance/charter.md

```diff
@@ -0,0 +1,57 @@
+Governance Committee Charter
```

Comment:
Did any changes take place to this document? GitHub is not recognizing it as a rename, so I can't tell if it was renamed _and_ modified.

Reply:
At the time you asked this question, I think I had made some minor changes. But as of the latest revision, I have made considerable changes. One thing I'd like to call out, which was in the original and remains in this version, is that the CGC charter explicitly says that it is not the place for technical decisions to be made. I've attempted to clarify that _a little bit_ in this draft, by referencing SIGs and WGs.